data/mods/Dark-Skies-Above/overrides/items/carnivore.json

```diff
@@ -27,22 +27,22 @@
     "id": "bone_tainted",
     "category": "other",
     "copy-from": "bone_tainted",
-    "name": "alien bone",
+    "name": { "str": "alien bone" },
     "description": "A piece of a hard material with many crests and protrusions, made of a hard material that occasionally leaves a bit of moisture on your hand when you touch it.  It seems basically like a bone, though not one you'd see in any anatomy textbook.  You could eat it, but no doubt it would not only taste henious but also make you sick, being made of material your body was never supposed to injest.  It might at least be able to be used for something, if not as well as the earthly variant due to its fiddly size and shape."
   },
   {
     "type": "COMESTIBLE",
     "id": "fat_tainted",
     "category": "other",
     "copy-from": "fat_tainted",
-    "name": "alien fat",
+    "name": { "str": "alien fat" },
```

Comment:
And all these "xxx fat", "xxx tallow", "xxx meat", "xxx lard" should use "str_sp" too.